src/containers/RecipeInputPage/css/index.css

```diff
@@ -0,0 +1,141 @@
+.body {
+  background-color: white;
+}
+
+.main {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  /* scroll-behavior: none; */
+  /* overflow: hidden; */
+  text-decoration-color: white;
+}
+
+p {
```

Comment:
this 'p' is modifying the color of all the p tags across the whole project. Give it a class and modify it, not the whole selector